eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantEntity.java

```diff
@@ -47,22 +48,31 @@
     @Field("alt")
     private String alternateAllele;
 
+    @Indexed
+    @Field("rs")
+    private Long clusteredVariant;
+
     @Field("evidence")
     private boolean supportedByEvidence;
 
-    SubmittedVariantEntity() {
+    @Field("matchAsm")
+    private Boolean matchesAssembly;
```

Comment:
alternatively, we can move this field to ImportedSubmittedVariantEntity only and fill the getter in this class as:
```
    @Override
    public Boolean getMatchesAssembly() {
        return null;
    }
```
and continue assuming in other places that null means true

Reply:
you mean "store it only when it **doesn't** match the assembly" right?

Reply:
checked. it's not writtten, as we intend.

Reply:
> you mean "store it only when it doesn't match the assembly" right?

I was thinking only about the dbSNP variants, but given that all the EVA variants should match it, then you're right it's better to store it only when it doesn't match. Same for the alleles mismatch then.

The 'validated' flag is the only one that would be inconsistent, representing a positive instead of a negative trait, but again the number of validated ones is much lower so it makes sense in terms of storage optimization.

Reply:
I agree with storing just the flags that are less usual (matchAssembly=false, validated=true ...)

Reply:
I agree, storing the less common ones would be a better option in order to optimise

Reply:
ok, so applying to every flag the rule of not writing for the most common value, this are the defaults (used if the flag is missing):
- supportedByEvidence true 
- matchesAssembly true
- allelesMatch true
- validated false

although the first one won't be usually true for dbsnp, but will be for eva. I wouldn't support different defaults depending of eva or dbsnp because it might require more engineering to avoid confusion.

Reply:
I agree with these defaults.

Reply:
👍